plugins/source/aws/client/multiplexers.go

```diff
@@ -106,13 +123,31 @@ func ServiceAccountRegionScopeMultiplexer(service string) func(meta schema.Clien
 				l = append(l, client.withPartitionAccountIDRegionAndScope(partition, accountID, cloudfrontScopeRegion, wafv2types.ScopeCloudfront))
 				for region := range client.ServicesManager.services[partition][accountID] {
 					if !isSupportedServiceForRegion(service, region) {
-						meta.(*Client).Logger().Trace().Str("service", service).Str("region", region).Str("partition", partition).Msg("region is not supported for service")
+						if client.specificRegions {
+							skippedRegions = append(skippedRegions, region)
+						}
+						client.Logger().Trace().Str("service", service).Str("region", region).Str("partition", partition).Msg("region is not supported for service")
 						continue
 					}
 					l = append(l, client.withPartitionAccountIDRegionAndScope(partition, accountID, region, wafv2types.ScopeRegional))
 				}
 			}
 		}
+		generateLogMessages(client, table, service, skippedRegions, len(l) == 0)
 		return l
 	}
 }
+
+func generateLogMessages(client *Client, table, service string, skippedRegions []string, emptyMultiplexer bool) {
+	if len(skippedRegions) == 0 {
+		return
+	}
+	loggerEvent := client.Logger().Warn()
```

Comment:
I still not sure this needs to be a warning. Let's say I use `tables: ["*"]` and specify a region, that is supported on some tables, and not supported on others. I'll get the warning, but the only way to fix it is to split the config correct? We don't have a way to skip regions for a specific table.
If that's the only way to fix the warning, I'd make this informational.

I agree that if we detect that a table is skipped altogether we should print an error

Reply:
To me skipping a table is the same level as skipping an account... and we log a `warn` entry when an account is skipped

Reply:
But we're not skipped a table altogether. We only skip it for some regions where

Reply:
For 1. I don't think there's a way to fix it other than splitting the config. For 2. you can simply skip the table

Reply:
Changed it to `Info` and `Warn`